gh-actions/start.sh

```diff
@@ -8,9 +8,10 @@ if [ ! -d "$folder" ] ; then
           context: ../../../../
         ports:
             - 8082:8080
-        environment: 
+        environment:
             TEUTHOLOGY_API_SERVER_HOST: 0.0.0.0
             TEUTHOLOGY_API_SERVER_PORT: 8080
+            TEUTHOLOGY_API_SQLITE_URI: sqlite:////teuthology_api/teuthology.db
```

Comment:
This should also be documented in README.
Also, is this the entire change that's needed for setting up db in docker setup? When testing with postgres, I remember I had to start postgres as a new docker service like https://github.com/ceph/teuthology-api/pull/19/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R42. Is that not needed here?

Reply:
no, SQLite won't require any additional setup as it is an embedded, file-based DBMS that can be used in python applications without any dependencies